scripts/rollup/cleanup-plugin.js

```diff
@@ -37,10 +36,7 @@ function cleanup(opts = {}) {
       }
 
       return transform(code, {
-        plugins: [
-          !opts.maintainImports && [babelPluginModularGraphQL, { extension: opts.extension }],
-          removeEmptyImports
-        ].filter(Boolean),
+        plugins: [removeEmptyImports],
```

Comment:
Should we still keep the plugin around for CommonJS bundles?

Reply:
Yes, rollup code splitting still seems to have some empty stuff :(